prez/reference_data/search_methods/search_jena_fulltext_name.ttl

```diff
@@ -9,6 +9,9 @@ prez:jenaFTName a prez:SearchMethod ;
     rdf:value """
         {
             SELECT * {(?search_result_uri ?weight ?match ?graph ?predicate) <http://jena.apache.org/text#query>
-            (<http://www.example.org/resources#NameProps> "$TERM")} LIMIT $LIMIT
+            (<http://www.example.org/resources#NameProps> "$TERM")
```

Comment:
Quick question with this NameProps predicate. Does this get replaced somewhere else or?

Reply:
No it must be defined in the Jena FT search index - I have used it to refer to a set of label predicates.
I've removed the Jena FT Search now to prevent any confusion.